src/PrestaShopBundle/Controller/Admin/CommonController.php

```diff
@@ -117,6 +117,13 @@ public function paginationAction(Request $request, $limit = 10, $offset = 0, $to
                 'limit' => '_limit'
             )
         ));
+        $jumpPageUrl = $this->generateUrl($routeName, array_merge(
+            $callerParameters,
+            array(
+                'offset' => 999999,
```

Comment:
For safety in case the route parameters evolves, to avoid JS construction of the URL that will be broken in this case. It's not the first use of this mechanism. Another should be to allow a special value on offset restrictions (\d+|_replace_me_) but this allow the route to be valid from the Router, with the _replace_me_ value that will crash in the controller.